packages/calcite-components/src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -1040,11 +1040,11 @@ describe("calcite-input-time-picker", () => {
         "--calcite-input-time-picker-shadow-focus": {
           selector: "calcite-input-time-picker",
           shadowSelector: "calcite-time-picker",
-          targetProp: "--calcite-time-picker-input-box-shadow-focus",
```

Comment:
Noice! Can you also update the PR title to include fixing this and action's tests?